src/cpp/_pythia8.cpp

```diff
@@ -244,6 +245,8 @@ PYBIND11_MODULE(_pythia8, m)
         .def("vt", event_array_v<Vec4_tt>)
         .def("parents", event_array_parents)
         .def("children", event_array_children)
+        .def("reset", &Event::reset)
+        .def("append", py::overload_cast<int, int, int, int, double, double, double, double, double, double, double>(&Event::append), "pdgid"_a, "status"_a, "col"_a, "acol"_a, "px"_a, "py"_a, "pz"_a, "e"_a, "m"_a = 0, "scale"_a = 0, "pol"_a = 9.)
```

Comment:
Same point as above.

Reply:
I use only one of the versions of append and arguments to that overload are the same as in this interface.